webpack.config.js

```diff
@@ -164,6 +164,124 @@ module.exports = {
 				},
 			} ) )
 		),
+		new CopyWebpackPlugin( [
+			{
+				from: './packages/block-library/build-style/*/style.css',
+				test: new RegExp(
+					`([\\w-]+)${ escapeRegExp( sep ) }style\\.css$`
+				),
+				to: 'build/block-library/blocks/[1]/style.css',
+				transform: ( content ) => {
```

Comment:
Is `transform` function the same for all CSS copy targets? I can't spot any differences so it would be a good opportunity to extract a shared helper.

Reply:
Good point. Done in https://github.com/WordPress/gutenberg/pull/25220/commits/831e38a76a3504c1e11ccd06590fd3ce8ecd50ec :+1: